src/api/plans.ts

```diff
@@ -96,19 +108,68 @@ router.delete("/plans/:planId", async (req, res) => {
   res.send();
 });
 
-router.get("/plans/:planId/courses", (req, res) => {
-  res.send({
-    courseName: "string",
-    courseNumber: 4056,
-    units: 4,
-    crosslisted: 53,
-    prerequisites: [
-      {
-        needed: 3,
-        "courses:": [12, 52, 236],
-      },
-    ],
-    offered: ["F", "W", "SP", "SU"],
-    description: "course description",
+router.get("/plans/:planId/courses", async (req, res) => {
+  const planCourses = await sql<PlanInfo[]>`
+    WITH recursive R AS(
+    SELECT program_courses.course_id as course FROM plans
+    JOIN programs on plans.program_id = programs.id
+    JOIN program_courses ON programs.id = program_courses.program_id
+    WHERE plans.id = ${req.params.planId}
```

Comment:
Plans are per-user, so we should also be calling auth and verifying the requested plan belongs to the current user.

See the get plan endpoint as an example: https://github.com/DegreePrerequisiteTracker/Degree-Prerequisite-Tracker/blob/main/src/api/plans.ts#L60-L73